functests/cincinnati_creation_test.go

```diff
@@ -13,9 +14,25 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/klog"
 	client "sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// TestMain will perform test setup, exec each test, and perform test tear down.
+func TestMain(m *testing.M) {
+	setUp()
+	retCode := m.Run()
+	os.Exit(retCode)
+}
+
+// setUp performs test setup.
+func setUp() {
+	if err := routev1.AddToScheme(scheme.Scheme); err != nil {
```

Comment:
Is there a reason you dropped this?

https://github.com/openshift/cincinnati-operator/blob/2df239a8486d2ba3aa0d9925e5d505105ab36afe/functests/functests_suite_test.go#L16

Reply:
Yeah, we don't import that yet.  We can add it with the commit that adds the import

Reply:
But you're just adding back stuff that shouldn't have been removed to begin with. I should have caught it and commented in in the `Remove Ginkgp` commit so not understanding why this commit doen't just add it all back in.

Reply:
We [used to][1] add the route scheme (twice?).  And out local APIs, which you link.  I dunno why we don't actually need those schemes injected, but tests are green, so we must not need them.

[1]: https://github.com/openshift/cincinnati-operator/pull/69/files#diff-30ccb8be4f6b0bb46f097a71158dd8e4b22b959348fee463959aae62fb041d29L20